test/immutable-store-state.js

```diff
@@ -0,0 +1,40 @@
+import Alt from '../dist/alt-with-runtime'
+import Immutable from 'immutable'
+import { assert } from 'chai'
+import sinon from 'sinon'
+
+const alt = new Alt()
+
+const actions = alt.generateActions('fire')
+
+const assign = require('object-assign')
+
+const store = alt.createStore({
+  displayName: 'ImmutableStore',
+
+  state: Immutable.Map({}),
```

Comment:
Ah, I see that now in the docs. Thanks for setting me straight.  I still think it should be configurable despite whether or not we pick Record or Map as the default.